source/max-staleness/tests/ReplicaSetWithPrimary/Nearest3.json

```diff
@@ -0,0 +1,39 @@
+{
+  "heartbeatFrequencyMS": 25000,
+  "topology_description": {
+    "type": "ReplicaSetWithPrimary",
+    "servers": [
+      {
+        "address": "a:27017",
+        "type": "RSPrimary",
+        "avg_rtt_ms": 5,
+        "maxWireVersion": 5
+      },
+      {
+        "address": "b:27017",
+        "type": "RSSecondary",
+        "avg_rtt_ms": 21,
+        "maxWireVersion": 5
+      }
+    ]
+  },
+  "read_preference": {
+    "mode": "Nearest"
+  },
+  "suitable_servers": [
```

Comment:
With Nearest mode, isn't "a:27017" also a suitable server?  Java driver fails this test now because it thinks that it is.

Reply:
closing because for re-review